DESCRIPTION

```diff
@@ -1,9 +1,12 @@
 Package: iSubGen
 Title: Integrative Subtype Generation
 Version: 1.0.2
-Date: 2023-11-06
-Author: Natalie Fox
-Maintainer: Natalie Fox <nfox@mednet.ucla.edu>
+Date: 2024-08-23
+Authors@R: c(
+	person("Paul Boutros", role = "cre", email = "PBoutros@mednet.ucla.edu"),
+	person("Adriana Salcedo", role = "aut"),
+	person("Dan Knight", role = "ctb"),
```

Comment:
And you should probably put me last. I've only contributed to general package standardization stuff. Nothing functional.